src/validator/TraversalValidator.cpp

```diff
@@ -33,7 +33,11 @@ Status TraversalValidator::validateStarts(const VerticesClause* clause, Starts&
                    << "but was`" << type.value() << "'";
                 return Status::Error(ss.str());
             }
+
             starts.srcRef = src;
+            auto encode = src->encode();
+            auto decode = Expression::decode(encode);
+            startVidList_->add(decode.release());
```

Comment:
`src` is a column, we should collect first.

Reply:
done